src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -287,27 +287,38 @@ def convert_middle_name_to_initial(pop: pd.DataFrame) -> pd.DataFrame:
 
 
 def sample_acs_standard_households(
-    target_number_sims: int,
+    num_households: Optional[int],
     acs_households: pd.DataFrame,
     acs_persons: pd.DataFrame,
     randomness: RandomnessStream,
+    target_number_sims: Optional[int] = None,
```

Comment:
You shouldn't change the order of the parameters unless you really need to, and you don't in this case.